docs/ref/contrib/admin/index.txt

```diff
@@ -965,6 +965,22 @@ subclass::
     editing, or deleting an object. You can have filters cleared by setting
     this attribute to ``False``.
 
+.. attribute:: ModelAdmin.show_facets
+
+    .. versionadded:: 5.0
+
+    Set ``show_facets`` to ``True`` to have all filter options appended with
+    the count of results that match that filter option in brackets. These
+    counts will update in line with currently applied filters. By default, this
+    is set to ``False``.
+
+    .. admonition:: Performance considerations with facets
```

Comment:
```suggestion
    .. admonition:: Performance considerations with facets.
```
Does this need a full stop?

Reply:
A lot of the other admonitions don't have full stops so I guess maybe it's like a title and not needed? I'm not sure though, happy to add it in